src/components/CallButton.tsx

```diff
@@ -1,22 +1,22 @@
 import styled from 'styled-components';
-import { CTAFlex } from './CTA';
```

Comment:
If the CTAFlex is not been used again, make sure to get rid of unused code

Reply:
I've removed all refrences.